DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -95,42 +111,45 @@ ${FSLDIR}/bin/fslmaths ${datadir}/data -abs -Tmin -bin -fillh ${datadir}/fov_mas
 	 
 if [ ! $GdCoeffs = "NONE" ] ; then
     echo "Correcting for gradient nonlinearities"
-	# Note: "data_warped" is eddy-current and suspectibility distortion corrected (via 'eddy'), but prior to gradient distortion correction
-	# i.e., "data_posteddy_preGDC" would be another way to think of it
-    ${FSLDIR}/bin/immv ${datadir}/data ${datadir}/data_warped
+	  # Note: data in the warped directory is eddy-current and suspectibility distortion corrected (via 'eddy'), but prior to gradient distortion correction
+	  # i.e., "data_posteddy_preGDC" would be another way to think of it
+    mkdir -p ${datadir}/warped
```

Comment:
If we want to consolidate the code a bit, we could define `warpedDir=${datadir}/warped` and then use `${warpedDir}` throughout the following.